make.ps1

```diff
@@ -18,6 +18,11 @@ function FindMSBuild
 	return $null
 }
 
+function UtilityNotFound
+{
+	echo "OpenRA.Utility.exe could not be found. Build the project first using the `"all`" command."
```

Comment:
In Makefile logic this would just start the utility build automatically. We may switch to a proper task based build system like https://github.com/psake/psake in the future.